### PR TITLE
Add a structured logger to delphi utils

### PR DESCRIPTION
**Pull request by benjaminysmith**
### Description
Adds a structured logger to delphi utils

### Changelog
- Adds a logger file with utility method to configure and return a structured logger
- Add structlog to the manifest

### Fixes 
- Incremental progress toward #606

### CODE REVIEW

File: _delphi_utils_python/delphi_utils/logger.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import sys
+import structlog
+
+def get_structured_logger(name=__name__):
+    """Create a new structlog logger.
+    
+    Use the logger returned from this in indicator code using the standard 
+    wrapper calls, e.g.:
+
+    logger.warning("Error", type="Signal too low").
```
Comment (chinandrew) [inline _delphi_utils_python/delphi_utils/logger.py:12]: can we add a note on what the `name` arg is for, and also a line showing the initialization in this example?

**Review by chinandrew — COMMENTED** [review]
just one comment, rest looks fine

File: _delphi_utils_python/delphi_utils/logger.py (hunk 1 above)
Comment (benjaminysmith) [reply _delphi_utils_python/delphi_utils/logger.py:13]: Done.

**Review by chinandrew — APPROVED** [review]
:shipit: :shipit: :shipit:

**Pull request merged**